src/main/java/com/mineboundteam/minebound/magic/OffensiveSpells/EnderOffensiveSpell.java

```diff
@@ -34,17 +38,16 @@ public EnderOffensiveSpell(Properties properties, EnderOffensiveSpellConfig conf
 
     @Override
     public void use(ItemStack stack, InteractionHand usedHand, Level level, Player player) {
-        if (!level.isClientSide()) {
-            BlockHitResult result = (BlockHitResult) player.pick(teleportDistance, 1f, false);
-            BlockPos pos = result.getBlockPos().relative(result.getDirection());
-            double dX = Math.abs(player.getX() - pos.getX() - 0.5);
-            double dY = Math.abs(player.getY() - pos.getY());
-            double dZ = Math.abs(player.getZ() - pos.getZ() - 0.5);
-            // Only execute if player would move > 1 block
-            if (Math.floor(dX + dY + dZ) > 0) {
-                player.teleportTo(pos.getX() + 0.5, pos.getY(), pos.getZ() + 0.5);
-                reduceMana(manaCost, player);
-            }
+        BlockHitResult result = (BlockHitResult) player.pick(teleportDistance, 1f, false);
```

Comment:
Why remove the serverside check?

Reply:
Did it for the sound, but in hindsight, I'm going to revert it since teleporting is purely server-side.